### PR TITLE
Search for multiple roles

### PR DESCRIPTION
**Pull request by akazorg**
It would be handy having this feature.

Instead of:
```php
@if (
    Auth::user()->hasRole('roleA') || 
    Auth::user()->hasRole('roleB') || 
    Auth::user()->hasRole('roleC')
) ...
```

We can write:
```php
@if (Auth::user()->hasRole('roleA,roleB,roleC'))
```

### CODE REVIEW

**Comment by fletch3555** [comment]
I'd rather it pass in a list/array than a comma-delimited string.  Otherwise I'm completely onboard with this idea! Thanks!

**Comment by marktopper** [comment]
I must agree with @fletch3555.

I think it would be better to use this way:
```
@if (Auth::user()->hasRole('roleA'))
@if (Auth::user()->hasRole(['roleA', 'roleB']))
```

If you make those to an option, I will merge this in 👍

**Comment by akazorg** [comment]
Hello guys! great idea, I'll update it

**Comment by marktopper** [comment]
Why did you add the `routes/web.php` file to the pull request?

**Comment by marktopper** [comment]
Also just tested it and it works 👍

**Comment by akazorg** [comment]
Oh, sorry, It was a mistake, I was trying to figure out what happen. Thank's

**Comment by akazorg** [comment]
@marktopper is that ok? sry about the junk xD

**Comment by fletch3555** [comment]
@akaZorg, did you merge upstream/master into your branch right before making the change we requested? I think that's how these extra files keep getting in.  @marktopper, we might need to discuss how we're handling these to avoid this in the future.

**Comment by akazorg** [comment]
@fletch3555 i think that happened, because that commit log is relative to my local master branch -.-
Can we undo this? sorry for the mistake.

**Comment by marktopper** [comment]
It's okay, we will figure it out @akaZorg.

File: src/Traits/VoyagerUser.php [hunk 1]
```diff
@@ -25,6 +32,7 @@ public function setRole($name)
 
         if ($role) {
             $this->role()->associate($role);
+            $this->save();
```
Comment (marktopper) [inline src/Traits/VoyagerUser.php:35]: Please remove this, since it's no longer there in the `master` branch.

Comment (fletch3555) [reply src/Traits/VoyagerUser.php:35]: Hold off on that

**Comment by fletch3555** [comment]
@akaZorg,

I just fixed your branch by resetting back to before you pulled upstream/master in.  Your local branch will say it's behind some number of commits.  Please delete your local branch(or `reset --hard` your local branch to match).  Do NOT push your changes without doing this, or it will undo what I've done.

Thanks!

**Comment by akazorg** [comment]
Thanks guys, I'll reset local repo.

**Comment by fletch3555** [comment]
@akaZorg, I messaged you on Slack.  Please take a look and get back to me.

**Comment by akazorg** [comment]
Going to Slack

**Comment by marktopper** [comment]
Any progress on this?

**Comment by akazorg** [comment]
Thought you're gonna handle it.

**Comment by marktopper** [comment]
Cool, I will look into it ;)

**Comment by marktopper** [comment]
Seems like I can't make modifications to this pull requests for some reason.

I could go ahead and remake this in a new pull requests?

**Comment by akazorg** [comment]
Sent new PR for this, will close this.

**Pull request closed without merging**